src/bidiMapper/domains/context/cdpTarget.ts

```diff
@@ -38,34 +41,40 @@ export class CdpTarget {
     cdpClient: CdpClient,
     cdpSessionId: string,
     realmStorage: RealmStorage,
-    eventManager: IEventManager
+    eventManager: IEventManager,
+    preloadScriptStorage: PreloadScriptStorage
   ): CdpTarget {
     const cdpTarget = new CdpTarget(
       targetId,
       cdpClient,
       cdpSessionId,
-      eventManager
+      eventManager,
+      preloadScriptStorage
     );
 
     LogManager.create(cdpTarget, realmStorage, eventManager);
 
     cdpTarget.#setEventListeners();
 
-    // No need in waiting. Deferred will be resolved when the target is unblocked.
-    void cdpTarget.#unblock();
+    // No need in waiting.
+    // Deferred will be resolved when the target is unblocked.
+    cdpTarget.#unblock();
```

Comment:
please keep `void` if async function is not awaited intentionally

Reply:
Sure. In this case I removed it because `create` is not `async`, so `await` cannot be added anyway.